cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/BuiltInRolesAdministrationTest.scala

```diff
@@ -269,9 +269,12 @@ class BuiltInRolesAdministrationTest extends DocumentingTest with QueryStatistic
       section("How to recreate the `admin` role", "administration-built-in-roles-admin-recreate", "enterprise-edition") {
         initQueries(
           "DROP ROLE admin")  // setup so that later when the grant query gets executed, it will show systemUpdates: 1
+        note(p("In Neo4j 4.1.0 it is *not* possible to fully recreate a dropped admin role. " +
+          "Specifically, the ability to run procedures with the `@Admin` annotation like e.g. `dbms.listConfig` can *not* be restored." +
+          " Because of that, it is highly discouraged to drop the admin role."))
         p(
           """
-            |To restore the role to its original capabilities two steps are needed: First, if not already done, execute `DROP ROLE admin`.
+            |To restore the role to its original capabilities (minus the ability to run admin procedures) the  two steps are needed: First, if not already done, execute `DROP ROLE admin`.
```

Comment:
Seems like you either are missing something between `the   two` or that you should remove `the`